hack/verify-imports-order.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script verifies that all the imports have our preferred order.
+# see: https://github.com/kubernetes/kubeadm/issues/2515
+# Usage: `hack/verify-import-order.sh`.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+
+kube::golang::verify_go_version
+kube::golang::setup_env
+
+go install k8s.io/kubernetes/cmd/orderimports
+
+cd "${KUBE_ROOT}"
+
+ret=0
+$(kube::util::find-binary "orderimports") --include-path="cmd/kubeadm" --ignore-file="zz_generated" "$@" || ret=$?
```

Comment:
should we log an issue in kubernetes/kubernetes to gather feedback / discuss if we want to enable this in more packages?

have you tried running this with e.g. `--include-path="cmd/|pkg/"`?

Reply:
> have you tried running this with e.g. --include-path="cmd/|pkg/"?

pkg: 1534 / 3064 files need re-order
cmd: 102 / 507 files need re-order
too many files

> should we log an issue in kubernetes/kubernetes to gather feedback / discuss if we want to enable this in more packages?

I think it is necessary to let more people participate in the discussion

Reply:
@dims WDYT about logging the issue but waiting on discussion first before `help-wanted`?
...or perhaps you think we should not scope to the wider k/k.

Reply:
@neolit123 the latter ... let's get it solid here in kubeadm (run for some time and ensure we work out the kinks) before we open this to the rest of k/k.

Reply:
SGTM